gradle/libs.versions.toml

```diff
@@ -7,9 +7,9 @@ hamcrest = { strictly = '2.2' }
 zalando = { strictly = '0.27.1' }
 aws-lambda-java-core = { strictly = '1.2.3' }
 google-guava = { strictly = '33.0.0-jre' }
-karate-core = { strictly = '1.4.1' }
+karate-core = { strictly = '1.3.1' }
```

Comment:
Deprecating?

Reply:
1.4.1 had some issues with the IDE and did not recognize the feature files as valid and did not allow me to run them in the IDE

Reply:
Plugin for Karate ligger kanskje noe bak, eller er det kommet en oppdatering?